.github/workflows/publish-project.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: 2021 Alliander N.V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Publish Project
```

Comment:
Release project sound more logical for me.
We often use Publish for publishing to Github Packages / Docker Hub, might be confusing